source/_integrations/zwave_js.markdown

```
@@ -118,15 +119,15 @@ Use this My button:
 3. Wait for the installation to complete.
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)